subsys/bluetooth/audio/aics_client.c

```diff
@@ -616,7 +618,14 @@ static uint8_t aics_discover_func(struct bt_conn *conn, const struct bt_gatt_att
 			 */
 			sub_params->ccc_handle = attr->handle + 2;
 			sub_params->notify = aics_client_notify_handler;
-			bt_gatt_subscribe(conn, sub_params);
+			err = bt_gatt_subscribe(conn, sub_params);
+			if (err != 0 && err != -EALREADY) {
+				LOG_ERR("Failed to subscribe: %d", err);
+
+				inst->cli.cb->discover(inst, err);
```

Comment:
Shouldn't this be guarded, as is done further up in the function before usage too?
`if (inst->cli.cb && inst->cli.cb->discover) {`